drivers/windows/R/mounts.R

```diff
@@ -178,3 +180,46 @@ dide_locally_resolve_unc_path <- function(path, mounts = detect_mounts(),
   }
   unname(drop(mounts[i, "local"]))
 }
+
+path_on_linux <- function(path_dat) {
+  remap <- function(path_dat, server, dest) {
+    path_to_folder <- sprintf("\\\\\\\\%s\\\\", server)
+    if (grepl(path_to_folder, path_dat$path_remote)) {
+      inner_folder <- gsub(path_to_folder, "", path_dat$path_remote)
+      return(sprintf("/%s/%s/%s", dest, inner_folder, path_dat$rel))
+    }
+    FALSE
+  }
+
+  remap2 <- function(path_dat, server, folder, dest) {
+    path_to_folder <- sprintf("\\\\\\\\%s\\\\%s\\\\", server, folder)
+    if (grepl(path_to_folder, path_dat$path_remote)) {
+      inner_folder <- gsub(path_to_folder, "", path_dat$path_remote)
+      return(sprintf("/%s/%s/%s", dest, inner_folder, path_dat$rel))
+    }
+    FALSE
+  }
+
```

Comment:
Can this be documented and/or refactored so that it's more obvious what it is doing please?

Reply:
Done